src/libs/ui/src/atoms/Modal/modal.styles.css

```diff
@@ -0,0 +1,73 @@
+.modal-overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.25); 
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  padding: 1rem; 
+}
+
+.bg-brand-white-100 {
+  background-color: rgb(255, 255, 255);
+}
+
+.overflow-y-auto {
+  overflow-y: auto;
+}
+
+.w-\[var\(--sign-in-width-mob\)\] {
+  width: 25rem;
+}
+
+.h-\[var\(--sign-in-overlay-height-desktop\)\] {
+  height: var(--sign-in-overlay-height-desktop);
+}
+
+.relative {
+  position: relative;
+}
+.hidden span {
+  display: none;
+}
+
+.modal-body {
+  padding: 1rem;
+  text-align: left;
+}
+
+.modal-body h2 {
+  width: 100%; 
+  margin-bottom: 2rem; 
+  color: #000;
+  padding-top: 0;
+  font-size: 2rem; 
+  font-family: sans-serif;
+}
+
+.modal-footer {
+  padding: 1rem;
+  text-align: right;
+}
+
+button {
+  padding: 0.5rem 1rem;
+  font-size: 1rem;
+}
+
+.sign-in_close-icon__4BNoR {
+  background-image: url(https://www.pcaskinpro.com/home/_next/static/media/Close.194ef8fc.svg);
+  margin-top: 0.5rem;
+  height: 2rem;
+  width: 1.5rem;
+}
+h1.gigya-screen-caption {
+  margin-bottom: 32px !important;
+  color: #000 !important;
+  padding-top: 0 !important;
+  font-size: 32px;
+}
+
```

Comment:
Please check all unnecessary classes from modal.styles.css file, and file extension must be .scss